include/drivers/gsm_ppp.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2020 Endian Technologies AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef GSM_PPP_H_
+#define GSM_PPP_H_
+
+struct device;
+void gsm_ppp_start(const struct device *device);
+void gsm_ppp_stop(const struct device *device);
```

Comment:
Please write this
```
/** @cond INTERNAL_HIDDEN */
struct device;
void gsm_ppp_start(const struct device *device);
void gsm_ppp_stop(const struct device *device);
/** @endcond */
```
This way the document generation does not try to document these "private" functions.

Reply:
What about this change, could you apply it too.

Reply:
:+1: